ljepotaservis/ljepotaservis.Web/ClientApp/src/components/superAdminSide/AddStore/AddOwner.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+
+class AddStore extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      firstName: "",
+      lastName: "",
+      email: "",
+      username: "",
+      password: ""
+    };
+  }
+
+  handleChange = async event => {
+    await this.setState({ [event.target.name]: event.target.value });
```

Comment:
Is there a reason this class is async? Usually we do this only for I/O commands like fetch.

Reply:
Yes, because setState is async, so if I fire a callback right after setState without awaiting the callback will fire with the old state